lib/transaction/operation/operation_inflation_pf.go

```diff
@@ -0,0 +1,48 @@
+package operation
+
+import (
+	"encoding/json"
+
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/errors"
+)
+
+type InflationPF struct {
+	Amount       common.Amount `json:"amount"`
+	VotingResult string        `json:"voting-result"`
+}
+
+func NewInflationPF(
+	amount common.Amount,
+	votingResult string,
```

Comment:
`votingResult` has the operation hash of `CongressVotingResult`, right? The description in comment will be helpful.